pygeoapi/flask_app.py

```diff
@@ -205,7 +205,7 @@ def execute_process(name=None):
 @click.command()
 @click.pass_context
 @click.option('--debug', '-d', default=False, is_flag=True, help='debug')
-def serve(ctx, debug=False):
+def serve(ctx, server, debug=False):
```

Comment:
does passing `server` here do anything?  Should we document what it is in the docstring?